stable/prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -55,6 +55,18 @@ spec:
             - name: queries
               mountPath: /etc/config.yaml
               subPath: config.yaml
+    {{- if .Values.cloudSQL.enabled  }}
```

Comment:
I'd rather add an `extraContainer` here so it'll be easier to other users customise it.
Take a look on this [example](https://github.com/kubernetes/charts/blob/24e6370f09a3fbca58f2e761e7fba3fa2e702ec3/stable/elastabot/templates/deployment.yaml#L39)